### PR TITLE
Update readme to add development information

### PR DESCRIPTION
**Pull request by MatMoore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,11 +16,25 @@ update the spreadsheets.
...
+
+#### Setting up OAuth for Google's API
+
+If this step is skipped, the sidekiq jobs will make you follow the OAuth process every time
+you regenerate an inventory.
+
+1. Go to https://console.developers.google.com and create a project
+2. In Google API manager go to Credentials and add OAuth credentials with type "Other"
+3. Set environment variables and run the following rake task:
+    GOOGLE_CLIENT_ID=... GOOGLE_CLIENT_SECRET= bundle exec rake google:auth
```
Comment (jackscotti) [inline README.md]: minor neatpick: missing `...` after `GOOGLE_CLIENT_SECRET=`

File: README.md [hunk 2]
```diff
@@ -16,11 +16,25 @@ update the spreadsheets.
...
+4. Follow the instructions to get a GOOGLE_REFRESH_TOKEN. Note that this is a non-expiring token that will grant access to anything in your google drive.
+5. In API manager, search for Drive API and enable it.
+
+#### Running the application
```
Comment (jackscotti) [inline README.md]: ## rather than ####?

Comment (MatMoore) [inline README.md]: How about ### ?

File: README.md [hunk 3]
```diff
@@ -55,6 +69,6 @@ or if you wanted to deploy branch with name 'my-branch'
 [MIT License](LICENCE)
 
 
-## Google Authentication
+## More information
 
 See the wiki: https://gov-uk.atlassian.net/wiki/pages/createpage.action?spaceKey=FS&fromPageId=44761298
```
Comment (tijmenb) [inline README.md]: Can you remove this link while you're at it? Doesn't go anywhere.

Comment (MatMoore) [inline README.md]: I've replaced it with the actual link :link:

**Comment by Davidslv** [comment]
:+1:

**Pull request merged**